File: Data Structures & Algorithms/counting-bits/submission-0.java
class Solution {
    public int[] countBits(int n) {
        int res[] = new int[n+1];
        for(int i=0; i<=n; i++) {
            res[i] = countBitsForNht(i);
        }
        return res;
    }
    
    public int countBitsForNht(int nht){
        int res = 0;
        while (nht != 0) {
            res = res + (nht & 1);
            nht = nht >> 1;
        }
        return res;
    }
}

File: Data Structures & Algorithms/max-water-container/submission-0.py
class Solution:
    def maxArea(self, heights: List[int]) -> int:
        h = 0
        t = len(heights)-1
        res = 0
        while h < t:
            v = min(heights[h], heights[t]) * (t - h)
            res = max(res, v)
            if heights[h] > heights[t]:
                t -= 1
            else:
                h += 1
        return res
        